code/modules/mob/living/silicon/robot/robot.dm

```diff
@@ -535,6 +535,19 @@
 			toner = tonermax
 			qdel(W)
 			to_chat(user, "<span class='notice'>You fill the toner level of [src] to its max capacity.</span>")
+
+	else if(istype(W, /obj/item/device/flashlight))
```

Comment:
Borg lamps are made from flashes

Reply:
How are the flashes their eyes and their weapon which can burn out and also their lamp

Reply:
Two flashes are installed in the head. The cyborg uses one flash as a weapon, the other is used as a light source for the headlamp. (Also a camera, I guess?) It uses a low, constant flow of energy instead of storing it in a capacitor for release all at once.

Reply:
Can't do it because of borg flashing actually

Reply:
Flashes are somehow borgs eyes, its headlight, its weapon, and its greatest combat weakness